ssh/tailssh/incubator.go

```diff
@@ -233,6 +242,19 @@ func beIncubator(args []string) error {
 	return cmd.Run()
 }
 
+// subprocEnviron filters and returns subset of os.Environ.
+func subprocEnviron() []string {
+	all := os.Environ()
```

Comment:
Could we make this function use an allowlist? I'm scared of things getting injected into the environment by stuff like the dynamic loader, when we really only want to pass through SSH_CONNECTION and SSH_CLIENT and nothing else.